valkey.conf

```diff
@@ -1354,7 +1354,11 @@ lazyfree-lazy-user-flush yes
 #
 # prefetch-batch-max-size 16
 #
-# NOTE: If you want to test the server speedup using valkey-benchmark, make
+# NOTE:
+# 1. 'io-threads-do-reads' config is deprecated and will be removed in the future,
+# please avoid using this option if possible.
```

Comment:
Mention also that it has no effect? (Some deprecated features still work, but this one doesn't.)

```suggestion
# 1. The 'io-threads-do-reads' config is deprecated and has no effect. It 
will be removed in the future. Please avoid using this option if possible.
```

Reply:
We won't necessarily remove it in the feature, we shouldn't use such language.

Reply:
Sorry for merging too fast. Shall we just delete the sentence "will be removed in the future"? A follow-up PR is welcome.

Reply:
Sure @madolson, I have noted your point and created https://github.com/valkey-io/valkey/pull/1150 to revoke the statement from valkey.conf.